RecoLocalCalo/EcalRecProducers/python/ecalRecHit_cfi.py

```diff
@@ -72,12 +72,15 @@
     #  ~64 GeV is the TP saturation level
     logWarningEtThreshold_EB_FE = cms.double(50),# in EB logWarningThreshold is actually in E (GeV)
     logWarningEtThreshold_EE_FE = cms.double(50),# in EE the energy should correspond to Et (GeV) but the recovered values of energies are not tested if make sense
+    singleChannelRecoveryMethod = cms.string("BDTG"), #default NeuralNetworks
+    singleChannelRecoveryThreshold = cms.double(0.70),##default 8
+    sum8ChannelRecoveryThreshold = cms.double(0.),##default 8
```

Comment:
what is the purpose of these ```default xyz``` comments here?
You mentioned already that for the final PR the recovery will be disabled.
It is enough to change the value of recoverEBIsolatedChannels for that.

What is the plan for the NN recovery? Is it going to be abandoned or are you planning to have an option to support both?
If both need to be supported, there should be a process modifier or (less preferred) a python "customise" method to switch from one set of settings to another.

Reply:
@slava77   The NN ....  it has never been used since it was deployed. We do not have a special plan on it. We would like to hear from the ECAL DPG conveners before doing something. We will get back to you on this
@crovelli @zghiche @fcouderc

Reply:
@slava77 Hi Slava, as you might already read from the email thread shared with the ECAL DPG conveners, we can abandon the NN algorithm. Practically speaking, besides avoiding the upload of the algorythm, should we physically remove the whole software and related data ? @taroni

Reply:
Complete removal is welcome (including git rm NNfiles).
For this PR, please simply replace the NN with BDTG calls (in the current state of this PR it would mean removing the nn-related data members and function calls from the places that you have modified already in this PR.